src/ert/_c_wrappers/enkf/ert_config.py

```diff
@@ -548,25 +596,35 @@ def _workflows_from_dict(cls, content_dict, substitution_list):
 
         for hook_name, mode_name in hook_workflow_info:
             if mode_name not in [runtime.name for runtime in HookRuntime.enums()]:
-                raise ConfigValidationError(
-                    errors=[f"Run mode {mode_name!r} not supported for Hook Workflow"]
+                combined_error.add_error(
+                    ConfigValidationError(
+                        errors=f"Run mode {mode_name!r} not supported for Hook "
+                        f"Workflow"
+                    )
```

Comment:
Need to `continue` if this error is hit, otherwise `HookRuntime.from_string(mode_name)` will raise. Seeing as this did not fail the tests, we should probably add a test that would have caught this error. It would have to be a unit test because parsing also guards against this error. Still, it is not correct behavior for the `_workflows_from_dict` function.

Reply:
Nice catch. I did add the continue and will add the test that fails without the continue